src/tests/class_tests/openms/source/MRMDecoy_test.cpp

```diff
@@ -409,168 +396,17 @@ START_SECTION((void generateDecoys(OpenMS::TargetedExperiment & exp,
   traml.load(OPENMS_GET_TEST_DATA_PATH(in), targeted_exp);
 
   MRMDecoy decoys = MRMDecoy();
-  decoys.restrictTransitions(targeted_exp, min_transitions, max_transitions);
```

Comment:
restrictTransitions was removed from MRMDecoy. The implementation in MRMAssay should be used if needed. Decoys should always have the exact same transition numbers as targets, or be excluded if this is not possible.